packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.js

```diff
@@ -148,14 +152,24 @@ export default class QuanticBreadcrumbManager extends LightningElement {
   }
 
   formatDateRangeBreadcrumbValue(breadcrumb) {
+    const data = getFromStore(this.engineId, Store.facetTypes.DATEFACETS);
     return {
       ...breadcrumb,
+      label: data[breadcrumb.field]?.label,
       values: breadcrumb.values.map(range => ({
         ...range,
-        value: this.formatDateRange(range.value),
+        value: data[breadcrumb.field]?.format(range.value),
       }))
     };
   }
+  
+  formatFacetBreadcrumbValue(breadcrumb) {
+    const data = getFromStore(this.engineId, Store.facetTypes.FACETS);
```

Comment:
What happens if there is nothing registered in the store for a given facetType?

Reply:
when there is nothing registered in the store, the `label` will be empty. 
Now I change it : when is empty then return `field`